client/src/component/description.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { DescriptionBox, ToggleMore } from '../style.jsx';
+
+const reducedDesc = function(desc){
```

Comment:
Can you add some clarifying comments for this function?

Reply:
will do. until then, that function splits the description at the closest empty space before 210 characters, which is then displayed before the user presses 'More'